tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,39 +6,51 @@
    "source": [
```

Comment:
<p>That's a very useful addition!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1354/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>